examples/02-plot/strips-example.py

```diff
@@ -0,0 +1,27 @@
+"""
+.._strips_example:
+
+Strips Figure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This is the title shown in the gallery documentation rather than a cell title.
```suggestion
Triangle Strips
~~~~~~~~~~~~~~~
```

Reply:
Also, you can accept the commit suggestions rather than implementing this in a separate commit (with the commit suggestion button). You can batch accept with:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request